client/lib/plans/features-list.js

```diff
@@ -949,7 +939,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'All Business features' ),
 		getDescription: () =>
 			i18n.translate(
-				'Including the ability to upload plugins and themes, priority support, advanced monetization options, and unlimited premium themes.'
+				'Including the ability to upload plugins and themes, priority support and advanced monetization options.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.